packages/help-center/src/components/help-center-chat-history.tsx

```diff
@@ -55,7 +55,7 @@ const Conversations = ( {
 	return (
 		<>
 			{ conversations.map( ( conversation ) => {
-				const lastMessage = getLastMessage( { conversation } );
+				const { lastMessage, unreadMessages } = getLastMessageAndUnread( { conversation } );
```

Comment:
It feels like the name of that function is missing a bit. Since `getLastMessageAndUnreadMessages` is both long and repetitive, I would suggest a slightly different name:
```suggestion
				const { lastMessage, unreadMessages } = getLastAndUnreadMessages( { conversation } );
```